src/overlays/actors/ovl_Obj_Jgame_Light/z_obj_jgame_light.c

```diff
@@ -12,14 +12,20 @@
 
 #define THIS ((ObjJgameLight*)thisx)
 
+typedef enum {
+    /* 0x0 */ NONE,
+    /* 0x1 */ CORRECT,
+    /* 0x2 */ INCORRECT,
+} SIGNAL;
```

Comment:
Enum name should be PascalCase, and even if it is file-local we still prefix it with something related to the file.

Reply:
You are right about the PascalCase, i'll fix it up, but if you don't like the name, maybe you should make a suggestion yourself. As i told you before, i'm not good finding names for things, and believe me, i tried on this one.

Reply:
The names are fine, they just need to be easily identifiable and unique by just appending the name of the Actor before. For example the enum name would become `ObjJgameLightSignal`. Should also append to the enum members like `OBJJGAMELIGHT_CORRECT`.